src/parser/warlock/demonology/modules/pets/PetTimelineTab/TabComponent/PetTimeline.css

```diff
@@ -6,3 +6,11 @@
   box-shadow: 0 0 11px rgb(22, 226, 1);
   border-radius: 2px;
 }
+.spell-timeline .events .absorbed {
```

Comment:
This component has undergone serious reconstruction in #3061 , and I think greenish border would fit better since it's a positive effect.
![image](https://user-images.githubusercontent.com/5068584/54879019-a5e49e00-4e34-11e9-8d2c-e18f42f1c799.png)

For now, I'd prefer if you used `META_CLASSES.EMPOWERED` instead of "absorbed".

Reply:
yeah, I initially went with empowered, but it felt wrong to be empowered by the thing that empowers others :)

I am happy to update it tho.